projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.ts

```diff
@@ -53,6 +61,28 @@ export class QueryDetailsComponent implements OnChanges{
   public hasRespondedToQuery(): boolean {
     const isAwaiting = this.queryResponseStatus === undefined || this.queryResponseStatus === QueryItemResponseStatus.AWAITING;
 
+    const lastChild = this.query?.children?.[this.query.children.length - 1];
+    const isFollowUp = lastChild?.messageType === this.followUpQuery;
+    const isRespond = lastChild?.messageType === this.respondToQuery;
+
+    this.enableServiceSpecificMultiFollowups = this.abstractConfig.getEnableServiceSpecificMultiFollowups() || [];
+    const isMultipleFollowUpEnabled = this.enableServiceSpecificMultiFollowups.some((jurisdiction) => jurisdiction === this.caseNotifier?.cachedCaseView?.case_type?.jurisdiction.id);
+
```

Comment:
I guess you can do this, but you should be listening to the case notifier and setting a class attribute for the current jurisdiction. Won't make much practical difference in this case, but it's working against the way that the notifier should work